GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -202,11 +204,24 @@ private void OnGuiCreateBuildSelect()
             EditorGUILayout.Space();
             EditorGUILayout.BeginHorizontal();
             EditorGUILayout.LabelField("APK File Name", GUILayout.MinWidth(FieldMinWidth));
-            QuickDeployConfig.Config.apkFileName = EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName,
-                GUILayout.MinWidth(FieldMinWidth));
+            QuickDeployConfig.Config.apkFileName =
+                EditorGUILayout.TextField(QuickDeployConfig.Config.apkFileName, GUILayout.MinWidth(FieldMinWidth));
+            if (GUILayout.Button("Browse", GUILayout.Width(ShortButtonWidth)))
+            {
+                const string defaultApkFileName = "base.apk";
+                QuickDeployConfig.Config.apkFileName = EditorUtility.SaveFilePanel("Choose file name and location",
+                    Path.GetDirectoryName(defaultApkFileName),
```

Comment:
Is this always empty string? Could just use string.Empty directly.

Reply:
Resolved.
Realized just setting defaultApkName = "base.apk" creates a confusion. Set this to Path.GetFullPath("apk"). 
Like this, Path.GetDirectoryname(defaultApkName) will return the directory and path.GetFileName will return the file name. The behaviour does not change when you just pass "base.apk" since unity will realize this and infer the full path, but I think it makes more sense to pass it as a full path in first place.